src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -333,6 +333,7 @@ def test_create_and_delete_advanced_sequence(session):
         session.active_advanced_sequence = sequence_name
 
 
+@pytest.mark.skip(reason="Please refer the Bug:1667")
```

Comment:
Since the fix is trivial, I'd rather see a PR for that (don't look at precise text) than a skip. Note that this failure is introduced by upgrading drivers which I argue should be done in its own separate PR.

But if you insist, then the proper way to reference issues is:

```suggestion
@pytest.mark.skip(reason="Workaround for issue #1667")
```

GitHub convention is  \#1667 is a link to an issue.

Reply:
I suggested to disable the test since there might be other tests we want to make more robust in this regard and I'd rather do that in a separate PR.